HazardScout/Recruiting-Project
# Ryan jacobson Recruitment Project

### Steps to Test

- Enter email and ticket quantity and press submit
- New user will then be added to the table following the default ones provided.
- You will see on the tickets column, the total number of tickets will update.
- You can then press on the edit button under the actions section.
- A modal will then be revealed which will allow you to make changes to the current user.
- Attempt to add a number to the tickets that will make the total surpass 20.
- It will not let you proceed.
- You may also press cancel to show that nothing has been updated.
- You may then attempt to add a new guest and have the ticket quantity surpass 20, it will also not let you proceed.
- Lastly, you can press the "x" under actions to remove the guest which will also update the total amount of tickets sold.
- Additionally, every change made will be saved in localStorage, you can test by refreshing the page.
